charts/argo-workflows/Chart.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: v2
 appVersion: v3.4.8
+kubeVersion: ">=1.23.0-0"
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.31.0
+version: 0.32.0
```

Comment:
This PR may give users an impact, so I wonder this is a time to bump major version ...? 🤔

Reply:
I recently bumped the supported version on the Workflows end: https://github.com/argoproj/argo-workflows/pull/11267/files

So I guess the right time is to do this when this gets released. I have no idea if it'll go into the next patch or not. The release process is... unclear.

Reply:
Thank you for the info. :)
That's a good idea to wait. I turn this PR into draft until then.

Reply:
Actually - because my change was just CI and docs, it's already released, there's nothing to cherry pick into the next patch.

Any new PRs will be run against k8s 1.27 already. So I've stll managed to avoid your question, but made the potential answer even more complicated.

Reply:
Oh I see. 

> Any new PRs will be run against k8s 1.27 already 

Well, then it seems that we can move on this PR...? 👀